desktop/src/containers/Account/accountActionContainer.tsx

```diff
@@ -27,23 +28,25 @@ interface Props extends RouteComponentProps {
 interface State {
   isFullScreen: boolean;
   isLoading: boolean;
+  isElectron: boolean;
 }
 
 export class AccountAction extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
       isLoading: false,
-      isFullScreen: true
+      isFullScreen: true,
+      isElectron: isElectron()
     };
   }
   componentDidMount = () => {
     //REMOVE before deploy
     // this.props.history.push(`/${this.props.type}/${routes.ANALYZE}`);
-    const isFullScreen = window.electronAPI.is_fullscreen();
-    this.setState({
-      isFullScreen
-    });
+    // const isFullScreen = window.electronAPI.is_fullscreen();
```

Comment:
This should be un commented and just conditionally run based on `this.state.isElectron`